scripts.js

```diff
@@ -15,4 +17,65 @@ closeBtn.addEventListener('click', () => {
+  <h3 class="popup-title">Project name goes here</h3>
+  <ul>
+      <li><button class="html pop1" type="button">HTML/CSS</button></li>
+      <li><button class="ruby pop1" type="button">Ruby on Rails</button></li>
+      <li><button class="js pop1" type="button">JavaScript</button></li>
+  </ul>
+  <div class="carousel">
+      <div class="carousel__contenedor">
+          <button aria-label="anterior" class="carousel__anterior">
+              <i class="fa-solid fa-chevron-left"></i>
+          </button>
+          <div class="carousel__lista">
+              <div class="carousel__elemento"><img src="./images/pcprogect.png" alt="Project1"></div>
+              <div class="carousel__elemento"><img src="./images/pcprogect1.png" alt="Project2"></div>
+              <div class="carousel__elemento"><img src="./images/pcprogect2.png" alt="Project3"></div>
+              <div class="carousel__elemento"><img src="./images/pcprogect3.png" alt="Project4"></div>
+              <div class="carousel__elemento"><img src="./images/pcprogect4.png" alt="Project5"></div>
+              <div class="carousel__elemento"><img src="./images/pcprogect5.png" alt="Project6"></div>
+          </div>
+          <button aria-label="siguiente" class="carousel__siguiente">
+              <i class="fa-solid fa-chevron-right"></i>
+          </button>
+      </div>
+      <div role="tablist" class="carousel__indicadores">
+              <div class="carousel__indicador"><img src="./images/pcprogect.png" alt="Project1"></div>
+              <div class="carousel__indicador"><img src="./images/pcprogect1.png" alt="Project2"></div>
+              <div class="carousel__indicador"><img src="./images/pcprogect2.png" alt="Project3"></div>
+              <div class="carousel__indicador"><img src="./images/pcprogect3.png" alt="Project4"></div>
+              <div class="carousel__indicador"><img src="./images/pcprogect4.png" alt="Project5"></div>
+              <div class="carousel__indicador"><img src="./images/pcprogect5.png" alt="Project6"></div>
+      </div>
+  </div>
+  <p class="popup-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+      sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+      <br><br>Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, 
+      consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.
+  </p>
+  <div class="see-buttons">
+    <button class="see-live">See live <img src="./icons/see-live.png"></button>
+    <button class="see-source">See source <img src="./icons/see-source.png"></button>
+  </div>
+  <div class="narrow-buttons">
+    <button class="previus"><img src="./icons/left.svg">Previus project </button>
+    <button class="next">Next project <img src="./icons/right.svg"></button>
+  </div>`;
+}
```

Comment:
- [ ] Great job implementing the popup window functionality, but as required the `projects` section should be created dynamically with the information from the projects `array`, that contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. Ie:

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```